internal/plugincommon/chainfee.go

```diff
@@ -0,0 +1,71 @@
+package plugincommon
+
+import (
+	"math/big"
+	"time"
+
+	"github.com/smartcontractkit/chainlink-ccip/internal/plugincommon/consensus"
+)
+
+type ChainFeeUSDPrices struct {
```

Comment:
q: is there a reason these types are in plugincommon and not in the chainfee package? Circular deps or something?

Reply:
cyclic dependency because I was returning it in ccip reader. On a second thought, I'm returning TimestampedBig and doing the conversion inside the processor for better encapsulation.